api/healthcheck_test.go

```diff
@@ -37,7 +37,7 @@ var _ = Describe("Healthcheck Handler", func() {
 
 	It("Should fail if redis failing", func() {
 		a := GetDefaultTestApp()
-		a.RedisClient.Client = GetFaultyRedis()
+		a.Leaderboards.RedisClient().Client = GetFaultyRedis()
```

Comment:
Should we expose the inner client and allow it to be changed like this?

Reply:
Makes sense that we don't. I'll change the test code to allow for the removal of this method.

Reply:
Great! Thanks